buildroot/share/fonts/u8glib-bdf2u8g.patch

```diff
@@ -45,3 +45,12 @@
      else if ( strcmp(p_buf, "BITMAP") == 0 )
      {
        bdf_line_state = BDF_LINE_STATE_BITMAP;
+@@ -1290,7 +1307,7 @@
+   fprintf(out_fp, "  Max Font    ascent =%2d descent=%2d\n", bdf_char_max_ascent, bdf_char_min_y);
+ 
+   fprintf(out_fp, "*/\n");
+-  fprintf(out_fp, "#include \"u8g.h\"\n");  
++  fprintf(out_fp, "#include <U8glib.h>\n");  
```

Comment:
@yhfudev — Since compilers aren't finding "u8g.h" in the local path this seems like the proper correction, but maybe it's supposed to be `"clib/u8g.h"` instead? See #10395.